apps/console/src/features/roles/components/edit-role/edit-role-updated.tsx

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Let's update the license header

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4290/commits/40c5244008b0c41e56b06852cec9b531cdf7b110